wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+---
+
+![match history team][Multi_mh_team image]
+
+_Example of Match History (Team)._
+
+## Team Mode
+
+### General
+
+#### The `Esc` key
+
+In general, pressing the `Esc` (Escape) key in keyboard now _does not pause the match_; in fact, it will trigger a warning at the lower-right to press `Esc` key again to quit.
+To quit the match and return to lobby, press it _again_.
+
+#### Visual Settings
+
+Some time given to use the Visual Settings below the beatmap during the loading phase of the beatmap.
+If Free Mods was used, the mods used applies to per player's interface and score differences may occur depending on mods used.
+Other players' interface may vary.
+
+#### Health bar
+
+When a player's health bar has been depleted completely, the player's beatmap play is considered failed.
+Entering failed state at least once will consider the score and accuracy of this match to not be counted as ranked entry.
+However, the failed player can continue playing and can be revived upon reaching full health.
+
+If [Sudden Death (SD)][Sudden Death wikilink]/[Perfect (PF)][Perfect wikilink] was used, revival from failed state is impossible.
+If playing Tag Coop/Tag Team Vs, a failed state means immediate match set and to the results screen.
+
+A failed or quitted player will be placed at the bottom of the leaderboard separately from other alive players, and placement changes can still occur among failed/quitted players.
+The failed players will not contribute towards the team score until revived back upon reaching full health.
+
+#### Result, Retry, and Replay
+
+Despite no online score leaderboard below the results screen, ranked score will still be counted as legitimate ranked entry.
+Result will not be shown at the Local leaderboard.
+The only way to see it is by Online Ranking leaderboard (assuming this score is higher).
+
+**No direct retry allowed** but the replay can be exported by pressing `F2` key in keyboard (will not contain any Multi-play elements).
+Do note that this does not apply to Tag Coop and Tag Team Vs (both are unrankable), where score will not be recorded and replay cannot be saved.
+
+#### Multi's leaderboard
+
+The leaderboard now shows the players in the match with live placements changes based on Win Condition as shown below:-
+
+**Head to Head / Team Vs:-**
+
+- If Score/Score v2: Score, hit score and combo multiplier are shown in real-time.
+- If Accuracy: Accuracy, hit score and combo multiplier are shown in real-time.
+- If Combo: Combo count are shown in real-time. Maximum combo count not shown.
+
+**Tag Coop / Tag Team Vs:-**
+
+- If Score/Score v2: Team's score and hit score are shown in real-time.
+- If Accuracy: Team's accuracy percentage and hit score are shown in real-time.
+- If Combo: Team's current combo count, score and hit score are shown in real-time.
+- If _Failed_: Match ends and (Tag Team Vs only) automated win for the surviving team.
+- If _Quit_: [Auto][Auto wikilink] will take over the _Quit_ player.
```

Comment:
I am not Auto and I have no idea. Also not sure if the collective spinner bonuses are added to the score or not.